tests/frontend/test_kibana_app/cypress/integration/pageobjects/settings/modules.page.js

```diff
@@ -0,0 +1,10 @@
+export const amazonAWSToggleButton = 'react-component > div:nth-child(1) > div > .euiFlexGroup .euiFlexGroup:nth-child(3) .euiSwitch__button';
+export const gCPToggleButton = 'react-component > div:nth-child(1) > div > .euiFlexGroup .euiFlexGroup:nth-child(4) .euiSwitch__button';
+export const openSCAPToggleButton = 'react-component > div:nth-child(2) > div > .euiFlexGroup .euiFlexGroup:nth-child(4) .euiSwitch__button';
+export const cisCatToggleButton = 'react-component > div:nth-child(2) > div > .euiFlexGroup .euiFlexGroup:nth-child(5) .euiSwitch__button';
+export const virusTotalToggleButton = 'react-component > div:nth-child(3) > div > .euiFlexGroup .euiFlexGroup:nth-child(3) .euiSwitch__button';
+export const osqueryToggleButton = 'react-component > div:nth-child(3) > div > .euiFlexGroup .euiFlexGroup:nth-child(4) .euiSwitch__button';
+export const dockerListenerToggleButton = 'react-component > div:nth-child(3) > div > .euiFlexGroup .euiFlexGroup:nth-child(5) .euiSwitch__button';
+export const gDPRToggleButton = 'react-component > div:nth-child(4) > div > .euiFlexGroup .euiFlexGroup:nth-child(3) .euiSwitch__button';
```

Comment:
Do these selectors belong to this view?
![image](https://user-images.githubusercontent.com/49887871/121225192-a9bb3a00-c85f-11eb-8330-5915b8549379.png)

Reply:
nope, those selectors are in settings,
this view:
![image](https://user-images.githubusercontent.com/23361539/121229408-45e74000-c864-11eb-95d7-0edf28f50b69.png)